test/e2e/data/infrastructure-metal3/cluster-template-centos-md-remediation/kustomization.yaml

```
@@ -0,0 +1,7 @@
+resources:
+- ../bases/ippool
+- ../bases/centos-kubeadm-config
+
+- mhc.yaml
+patchesStrategicMerge:
+  - md.yaml
```

nit: use consistent indent
```suggestion
- md.yaml
```

---

done